config.yaml

```diff
@@ -1,9 +1,9 @@
 alerts:
   matrix:
-    username: "rbgbot"
-    password: "password"
-    homeserver: "matrix.org"
-    roomId: "!id:matrix.org" # Obtained in Element by "Room Settings" -> "Advanced" -> "Internal Room ID"
+    homeserver: matrix.org
+    password: password
+    roomid: '!id:matrix.org'
+    username: rbgbot
```

Comment:
Maybe we want to keep the comment. There are different way to get something called `roomId`, but only copying like described in the comment works.

Reply:
Sure! This got lost when the config was written after generating the key.